app/templates/eventView.html

```diff
@@ -81,62 +127,68 @@ <h3><br>Description</h3><p>{{eventData.description|urlize}}</p><br>
-                                              <p><b>{{training}}</b>: Upcoming</p>
-                                          {% endif%}
-                                      {% endif %}
-                                  {% endfor %}
-                            {% endif %}
-                          </div>
-                      </div>
-            </div>
-            {% if filepaths.keys()|count %}
-            <div class="form-group mb-5">
-                <label class="mb-2" for= "eventAttachments"><strong>Event Attachments</strong></label>
-                <table>
-                    {% for key, value in filepaths.items() %}
-                        <tr id="attachment_{{value[1]}}" class="attachmentRow_{{key}}" data-id= "{{key}}">
-                            <td class="pr-5">
-                                <a class="mr-5" href="{{value[0]}}" target="_blank">{{key}}</a>
-                            </td>
-                        </tr>
-                    {% endfor %}
-                </table>
             </div>
-            {% endif %}
-            <div class="rsvp-btn text-center">
-                {% if (not isPastEvent) and (not isNewEvent)%}
-                    {% if userHasRSVPed %}
-                        <button type="submit" class="btn btn-danger" value="{{eventData.id}}" id="removeRsvpBtn">Remove RSVP</button>
-                    {% elif eventData.isRsvpRequired %}
-                        {% set buttonName = "Join Waitlist" if (eventData.rsvpLimit) and (currentEventRsvpAmount[eventData.id] >= eventData.rsvpLimit) else "RSVP" %}
-                        <button type="submit" class="btn btn-primary" value="{{eventData.id}}" id="rsvpBtn">{{buttonName}}</button>
+
+            <div class="col-md-6">
+                <div class="col-sm">
+                    {% if eventData.contactName and eventData.contactEmail %}
+                        <br><h3>Contact Information:</h3>
+                        <div>{{eventData.contactName}} - {{eventData.contactEmail}}</div>
+                    {% endif %}
+
+                    {% if program or not eventData.isAllVolunteerTraining %}
+                        <h3><br>Program Trainings</h3>
+                        {% for training, participated in programTrainings.items() %}
+                            {% if g.current_user.isCeltsAdmin %}
+                            <p>{{training}}</p>
+                            {% else %}
+                            {% if participated == 1 %}
+                                <p><b>{{training}}</b>: Attended</p>
+                            {% elif participated == 0 %}
+                                <p><b>{{training}}</b>: Not attended</p>
+                            {% else %}
+                                <p><b>{{training}}</b>: Upcoming</p>
+                            {% endif%}
+                                {% endif %}
+                        {% endfor %}
+                    {% endif %}
```

Comment:
Please make sure all your indentation is correct. It is not needed to run but it will help when someone is working in this section in the future.